.env.lacchain

```diff
@@ -71,4 +71,7 @@ REACT_APP_EOS_USE_BP_JSON_ON_CHAIN=false
 REACT_APP_EOS_BP_JSON_ON_CHAIN_CONTRACT=producerjson
 REACT_APP_EOS_BP_JSON_ON_CHAIN_TABLE=producerjson
 REACT_APP_EOS_BP_JSON_ON_CHAIN_SCOPE=producerjson
-REACT_APP_TOKEN_SYMBOL=
\ No newline at end of file
+REACT_APP_TOKEN_SYMBOL=
+REACT_APP_EOS_MAINNET=https://mainnet.eosio.cr
+REACT_APP_JUNGLE_TESTNET=http://testnet.eosio.cr
```

Comment:
https://

Reply:
we should add these variables to the remaining `.env` files also

Reply:
done